pages/docs/reference/python/functions/create.mdx

```diff
@@ -151,6 +151,10 @@ The `create_function` decorator accepts a configuration and wraps a plain functi
     </Properties>
   </Property>
 
+  <Property name="idempotency" type="string">
```

Comment:
Is it also optional, like [in TS SDK](https://www.inngest.com/docs/reference/functions/create#inngest-create-function-configuration-trigger-handler-inngest-function)?